app/assets/src/components/views/discovery/DiscoveryView.jsx

```diff
@@ -608,13 +608,15 @@ class DiscoveryView extends React.Component {
     const { mapSidebarTab } = this.state;
     this.setState(
       {
+        currentDisplay: "table",
```

Comment:
Can we select projects from the map?

Reply:
yes this is referring to clicking on the project in the right pane table while in the map

Reply:
Shouldnt we continue on the map?

Reply:
Jenn determined that it should set to the table view instead https://jira.czi.team/browse/IDSEQ-1087

Reply:
cc @happyimadesignr

Reply:
👌